### ISSUE TITLE
Verifying new session gets stuck on "Incoming verification request, waiting for partner to confirm"

### ISSUE THREAD

**Issue by jryans** [vector-im/element-web#15225]
Extracted from comments in https://github.com/vector-im/element-web/issues/14970#issuecomment-685985483.

Some users (at least @Y0hy0h) are still seeing issues with verifying a new session between Element Web and Android. Web seems to get stuck on a spinner after verifying emoji, while Android seems to think everything is done.

![grafik](https://user-images.githubusercontent.com/11377826/92802289-ad945b80-f3b6-11ea-9e86-c58c76867e32.png)
![grafik](https://user-images.githubusercontent.com/11377826/92802375-c270ef00-f3b6-11ea-8d54-5a081e028ce9.png)

From previous debugging, I believe there may be a race in the various state machines on the web side.

**Comment by Y0hy0h** [comment]
A friend of mine could not reproduce the issue. I tried again on my laptop (never tried it there before) where it did not hang, and I just checked on my Desktop (where the screenshots come from) and it works there too, now.

I noticed that the Android app now allows me to scan the QR code which was not the case before (I had only access to text or emoji). Maybe I received an update in the meantime that fixed the issue, or it was some caching problem.

I consider this problem fixed now. Thanks for your effort!
